lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -138,7 +138,6 @@ def __init__(self):
         self.machine_id = None
         self.enclosure_id = None
         self.mode = None
-        self.cvg_count = -1
```

Comment:
Should we set this to `self.cvg_count = 0`

Reply:
Not required anymore, it handled on line no 505